src/Features/ERDDAP/Map/index.tsx

```diff
@@ -170,11 +170,16 @@ export const ErddapMapBase: React.FC<BaseProps> = ({ platforms, platformId, heig
  * Map that is focused on the Gulf of Maine with the selected platform highlighted
  */
 export const ErddapMap: React.FC<Props> = ({ platformId, height }: Props) => {
-  return (
-    <UsePlatforms>
-      {({ platforms }) => <ErddapMapBase platforms={platforms} platformId={platformId} height={height} />}
-    </UsePlatforms>
-  )
+  const { isLoading, data } = usePlatforms()
+  const [isClient, setIsClient] = useState(false)
+
+  useEffect(() => {
+    setIsClient(true)
+  }, [])
+
+  if (data?.features && isClient) {
+    return <ErddapMapBase platforms={data?.features} platformId={platformId} height={height} />
+  }
```

Comment:
Do we need to `return null` as a drop through case?

Reply:
Yes, thanks for catching this